include/aws/mqtt/client.h

```diff
@@ -215,6 +212,7 @@ struct aws_mqtt_connection_options {
     struct aws_byte_cursor client_id;
     uint16_t keep_alive_time_secs;
     uint32_t ping_timeout_ms;
+    uint16_t publish_timeout_secs;
```

Comment:
can we use milliseconds instead?

Reply:
Sure, I just don't want to the timeout to be too short.